lib/versions.js

```diff
@@ -14,9 +14,9 @@ function newVersionFilter(opts) {
         // look for anything closer to the main release.
         if (opts.channel) {
             return (opts.channel === version.channel &&
-                    semver.rcompare(version.tag, opts.tag) < 0);
+                    semver.rcompare(version.tag, opts.tag) <= 0);
         } else {
-            return semver.satisfies(version.tag, '>' + opts.tag);
+            return semver.satisfies(version.tag, '>=' + opts.tag);
```

Comment:
These changes are required because Squirrel.Windows checks for equality locally.  It never expects a 204 like Squirrel.Mac does, but instead that you return all the version info, and it will check to see if there is a newer version available.   We already are checking to see if the tag is equal to the given tag in the osx update handler function, so this will be fine.